docs/tutorial.rst

```diff
@@ -136,19 +136,54 @@ variable in Python.  The same is true for deleting them too.
 
 Very nice.
 
-__xonsh_env__
+Environment Lookup with ``${}``
```

Comment:
i liked that flow, you talk about envvars with `$NAME` then you introduce something better and the last example leads to ellipsis which then you exlain. but np! i'll just reverse this two sections.